examples/aggregations/package.json

```diff
@@ -12,7 +12,7 @@
     "codegen": "graph codegen",
     "create": "graph create example --node https://api.studio.thegraph.com/deploy/",
     "create-local": "graph create example --node http://127.0.0.1:8020",
-    "deploy": "graph deploy example --ipfs https://api.thegraph.com/ipfs/ --node https://api.studio.thegraph.com/deploy/",
+    "deploy": "graph deploy example --ipfs https://ipfs.thegraph.com/ipfs/ --node https://api.studio.thegraph.com/deploy/",
```

Comment:
This (and all other package.json's below) should use `--ipfs https://ipfs.thegraph.com`